app/lib/__tests__/cloneutils-test.js

```diff
@@ -1,5 +1,5 @@
 import { expect } from "chai"
-import { clone } from "../cloneutils.js"
+import { clone } from "../cloneutils"
```

Comment:
Similarly, you don't _need_ to add the extension here when importing - it gets inferred to a JS or JSX file just fine.

We're not consistent with this, so I cleaned up all these extensions while I was in here.